bom/pom.xml

```diff
@@ -37,24 +39,30 @@
         <jackson-bom.version>2.14.3</jackson-bom.version>
         <typesafe-config.version>1.4.2</typesafe-config.version>
         <ssl-config-core.version>0.6.1</ssl-config-core.version>
-        <akka-bom.version>2.6.21</akka-bom.version>
-        <akka-persistence-mongo.version>3.0.8</akka-persistence-mongo.version>
-        <akka-http-bom.version>10.2.10</akka-http-bom.version>
-        <akka-management.version>1.1.4</akka-management.version>
-        <akka-stream-kafka.version>3.0.1</akka-stream-kafka.version>
         <kafka-client.version>3.0.2</kafka-client.version>
         <hivemq-mqtt-client.version>1.3.0</hivemq-mqtt-client.version>
         <sshd.version>2.9.2</sshd.version>
         <eddsa.version>0.3.0</eddsa.version>
         <lz4-java.version>1.8.0</lz4-java.version>
 
-        <!-- Keep these version consistent with akka-persistence-mongo.version's build.sbt -->
+        <pekko-bom.version>1.0.1</pekko-bom.version>
+        <pekko-http-bom.version>1.0.0</pekko-http-bom.version>
+        <pekko-persistence-mongodb.version>1.0</pekko-persistence-mongodb.version>
+        <pekko-persistence-inmemory.version>1.0.0</pekko-persistence-inmemory.version>
+        <pekko-management.version>1.0.0</pekko-management.version>
+        <pekko-connector-kafka.version>1.0.0</pekko-connector-kafka.version>
+        <parboiled.version>2.5.0</parboiled.version>
+
+        <!--  Necessary for pekko-persistence-mongodb -->
+        <metrics4-scala.version>4.2.9</metrics4-scala.version>
+
+        <!-- Keep these version consistent with pekko-persistence-mongodb.version's build.sbt -->
         <mongo-java-driver.version>4.3.4</mongo-java-driver.version>
```

Comment:
@alstanchev could you upgrade to the Mongo driver version pekko-persistence-mongo uses?

Reply:
Now we can also update to `pekko-persistence-mongodb` 1.0.1 :)